src/components/Patient/PatientRegistration.tsx

```diff
@@ -307,558 +288,476 @@ export default function PatientRegistration(
-              {t("name")}
-              <span className="text-red-500">*</span>
-            </Label>
-            <Input
-              {...fieldProps("name")}
-              placeholder={t("type_patient_name")}
-            />
-            <div className="mt-1" data-input-error>
-              {errors["name"] &&
-                errors["name"].map((error, i) => (
-                  <div key={i} className="text-red-500 text-xs">
-                    {error}
-                  </div>
-                ))}
-            </div>
-            <br />
-            <Label className="mb-2">
-              {t("phone_number")}
-              <span className="text-red-500">*</span>
-            </Label>
-            <Input
-              {...fieldProps("phone_number")}
-              onChange={(e) => {
-                if (e.target.value.length > 13) return;
-                setForm((f) => ({
-                  ...f,
-                  phone_number: e.target.value,
-                  emergency_phone_number: samePhoneNumber
-                    ? e.target.value
-                    : f.emergency_phone_number,
-                }));
-              }}
+
+        <Form {...form}>
+          <form
+            className="md:w-[500px] space-y-10"
+            onSubmit={form.handleSubmit(onSubmit)}
+          >
+            <PLUGIN_Component
+              __name="PatientRegistrationForm"
+              form={form}
+              patientId={patientId}
             />
-            <div className="mt-1" data-input-error>
-              {errors["phone_number"] &&
-                errors["phone_number"]?.map((error, i) => (
-                  <div key={i} className="text-red-500 text-xs">
-                    {error}
-                  </div>
-                ))}
-            </div>
 
-            <div className="mt-1 flex gap-1 items-center">
-              <Checkbox
-                checked={samePhoneNumber}
-                onCheckedChange={() => {
-                  const newValue = !samePhoneNumber;
-                  setSamePhoneNumber(newValue);
-                  if (newValue) {
-                    setForm((f) => ({
-                      ...f,
-                      emergency_phone_number: f.phone_number,
-                    }));
-                  }
-                }}
-                id="same-phone-number"
+            <div id="general-info" className="space-y-6">
+              <div>
+                <h2 className="text-lg font-semibold">
+                  {t("patient__general-info")}
+                </h2>
+                <div className="text-sm">{t("general_info_detail")}</div>
+              </div>
+
+              <FormField
+                control={form.control}
+                name="name"
+                render={({ field }) => (
+                  <FormItem>
+                    <FormLabel>
+                      {t("name")}
+                      <span className="text-red-500">*</span>
+                    </FormLabel>
```

Comment:
form label component has the required prop 😅 

```suggestion
                    <FormLabel required>
                      {t("name")}
                    </FormLabel>
```